### ISSUE TITLE
Support for Flood torrent client

### ISSUE THREAD

**Issue by Casuallynoted** [Casvt/Kapowarr#130]
**Describe the solution you'd like**
I'd love to see support for Flood added in addition to the newly added support for Qbittorrent!

**Comment by Casvt** [comment]
Yeah for sure. Other clients like rTorrent and deluge are on the planning too.

**Comment by Casvt** [comment]
Only now have I bothered to check the actual application out. It seems like it's just a web-UI wrapper around applications like rTorrent, qBittorrent and Transmission, not a torrent client itself. So shouldn't I just add support for those applications then?

**Comment by jangrewe** [comment]
@Casvt What's the ETA on Deluge support? Because that's the only thing holding me back, as i use Deluge for ... well, everything. 😁
